bin/add_frontend_server.rb

```diff
@@ -42,6 +46,10 @@ def execute_ssh_on_channel(channel, command, exit_error_string)
   opts.on('-h', '--help', 'Print this') { puts options; exit }
 end.parse!
 
+unless ['production', 'adhoc'].include?(options['environment'])
+  throw 'Environments other than production are currently unsupported'
```

Comment:
Does adhoc in fact produce a correctly running server?  (I seem to recall I had to take some additional manual steps to get adhoc working, like creating and seeding the local mysql server.)